.github/workflows/gen.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Run Node Client Generator and Create Pull Request
+on:
+  schedule:
+    # Run at the end of every day
```

Comment:
Just to confirm, we're running this every day instead of running when there are changes in google-cloudevents or this repos generation scripts because that would be too complex to implement? Or is there another consideration?

Reply:
Correct. This action runs on a cron within this repo. Most cron runs will not produce a PR.

The futher improvement, if technically possible, would be to run this action on a push event to the other repo (google-cloudevents). However, that desired behavior is not possible because cross-repo authorization requires creating a personal access token.

Maybe @averikitsch has an example of cross-repo solution, although that could be an improvement over cron.

Reply:
This is a really interesting pattern with a handful of moving parts that would be best to solidify. Do you have a design doc? Is there security reasons or other reasons that dissuade you from going the PAT route? I will create a doc with some thoughts

Reply:
As for notes on a design, there are notes in a doc unmarshal autogen.

Creating/maintaining a PAT secret won't be necessary with this approach. It should be clearer than other approaches. Any suggestions are welcome.

Reply:
The unmarshal autogen doc and meetings earlier this year was around this. I'm happy to answer more questions, I think chatting more on VC may be faster, feel free to ping me.

I'm not concerned about running an action every day. If we want to change to running every week, that is fine with me.

I hope we can make progress here, like we've done with other actions with the Workflows repo and the main Google CloudEvents repo.